wallpaper-bindings/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "wallpaper-bindings"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+more-wallpapers = {version = "0.3.1", features = ["fallback"]}
+
+[lib]
+name = "wallpaper-bindings"
+path = "src/lib.rs"
+
+[[bin]]
+name = "wallpaperbin"
+path = "src/main.rs"
```

Comment:
nit but there should be a newline char at end of file, I think it's a Windows thing that its not added by default.